package.json

```diff
@@ -72,6 +72,7 @@
     "litdoc": "1.5.6",
     "markdown-toc": "1.1.0",
     "mocha": "3.4.2",
+    "mock-fs": "^4.4.2",
```

Comment:
Can we specify a specific version instead of `^`? I know with `package-lock.json` this is less of an issue, but just to keep it consistent with how we set the other dependencies.